perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/registrartabs/ApplicationDetailTabItem.java

```diff
@@ -404,7 +404,7 @@ public void onClick(ClickEvent event) {
 		menu.addWidget(new HTML("<strong>STATE: </strong>"+Application.getTranslatedState(app.getState())));
 
 		GetApplicationDataById data = new GetApplicationDataById(appId);
-		data.setEditable(app.getState().equals("VERIFIED") || app.getState().equals("NEW"));
+		data.setEditable((app.getState().equals("VERIFIED") || app.getState().equals("NEW")) && !app.getType().equals("EMBEDDED"));
```

Comment:
This is not exactly, what we wanted. We don't want the embedded applications to be not editable. We want to disable editing items of type `EMBEDDED_GROUP_APPLICATION`.